### PR TITLE
#651: Fix gridConnector.js to clear cache on node-close…

### PR DESCRIPTION
**Pull request by wfischlein**
To make the fix for https://vaadin.com/pro/support#3047 work there is a fix in gridConnector.js to clear cache on node-close. To test it there is another button in TreeGridChangingHierarchyPage that adds subnodes to nodes ("b") that have already >0 subnodes.
To make that test compile https://github.com/vaadin/flow/pull/6202 has to be merged first

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-grid-flow/735)

### CODE REVIEW

File: vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js [hunk 1]
```diff
@@ -934,27 +943,27 @@ window.Vaadin.Flow.gridConnector = {
...
-                    e.detail.setDragData(type, grid.__selectionDragData[type]);
-                });
-            } else {
-                (grid.__dragDataTypes || []).forEach(type => {
-                    e.detail.setDragData(type, e.detail.draggedItems.map(item => item.dragData[type]).join('\n'));
-                });
-            }
+      if (grid._isSelected(e.detail.draggedItems[0])) {
+      // Dragging selected (possibly multiple) items
+      if (grid.__selectionDragData) {
```
Comment (tomivirkki) [inline vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js]: Lots of strange indentation changes in this file. Rather only include the relevant changes.

**Comment by Haprog** [comment]
Flow [2.1.0.alpha1](https://github.com/vaadin/flow/releases/tag/2.1.0.alpha1) is out now including vaadin/flow#6202. I guess this PR should be updated to depend on that Flow versions to make it run and then we can probably release this as `vaadin-grid-flow` 4.1.0.alpha1?

**Comment by web-padawan** [comment]
Hello @wfischlein, could you please check this PR again and update, or close if necessary?

Please note that Grid connector was updated so there are conflicts now.

**Comment by tomivirkki** [comment]
Closing as inactive. If still relevant, please re-apply to https://github.com/vaadin/vaadin-flow-components

**Pull request closed without merging**